loop.js

```diff
@@ -0,0 +1,5 @@
+const loopTriangle = () => {
+for (let i = '#'; i.length<=7; i += '#'){
+    console.log(i)
+}
+}
```

Comment:
Try to work on your indication and code formatting. Also ... no newline at the end of file :P